recipes/qt/6.x.x/conanfile.py

```diff
@@ -0,0 +1,553 @@
+            return {"clang": "freebsd-clang",
+                    "gcc": "freebsd-g++"}.get(str(self.settings.compiler))
+
+        elif self.settings.os == "SunOS":
+            if self.settings.compiler == "sun-cc":
+                if self.settings.arch == "sparc":
+                    return "solaris-cc-stlport" if self.settings.compiler.libcxx == "libstlport" else "solaris-cc"
+                elif self.settings.arch == "sparcv9":
+                    return "solaris-cc64-stlport" if self.settings.compiler.libcxx == "libstlport" else "solaris-cc64"
+            elif self.settings.compiler == "gcc":
+                return {"sparc": "solaris-g++",
+                        "sparcv9": "solaris-g++-64"}.get(str(self.settings.arch))
+        elif self.settings.os == "Neutrino" and self.settings.compiler == "qcc":
+            return {"armv8": "qnx-aarch64le-qcc",
+                    "armv8.3": "qnx-aarch64le-qcc",
+                    "armv7": "qnx-armle-v7-qcc",
+                    "armv7hf": "qnx-armle-v7-qcc",
+                    "armv7s": "qnx-armle-v7-qcc",
+                    "armv7k": "qnx-armle-v7-qcc",
+                    "x86": "qnx-x86-qcc",
+                    "x86_64": "qnx-x86-64-qcc"}.get(str(self.settings.arch))
+        elif self.settings.os == "Emscripten" and self.settings.arch == "wasm":
+            return "wasm-emscripten"
+
+        return None
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self, generator="Ninja")
+
+        self._cmake.definitions["INSTALL_MKSPECSDIR"] = os.path.join(self.package_folder, "res", "archdatadir", "mkspecs")
+        self._cmake.definitions["INSTALL_ARCHDATADIR"] = os.path.join(self.package_folder, "res", "archdatadir")
+        self._cmake.definitions["INSTALL_DATADIR"] = os.path.join(self.package_folder, "res", "datadir")
+        self._cmake.definitions["INSTALL_SYSCONFDIR"] = os.path.join(self.package_folder, "res", "sysconfdir")
+
+        self._cmake.definitions["QT_BUILD_TESTS"] = "OFF"
+        self._cmake.definitions["QT_BUILD_EXAMPLES"] = "OFF"
+
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.compiler.runtime == "MT" or self.settings.compiler.runtime == "MTd":
+                self._cmake.definitions["FEATURE_static_runtime"] = "ON"
+
+        if self.options.multiconfiguration:
+            self._cmake.generator = "Ninja Multi-Config"
+            self._cmake.definitions["CMAKE_CONFIGURATION_TYPES"] = "Release;Debug"
+        self._cmake.definitions["FEATURE_optimize_size"] = ("ON" if self.settings.build_type == "MinSizeRel" else "OFF")
+
+        for module in self._submodules:
+            if module != 'qtbase':
+                self._cmake.definitions["BUILD_%s" % module] = ("ON" if self.options.get_safe(module) else "OFF")
+
+        self._cmake.definitions["FEATURE_system_zlib"] = "ON"
+
+        # FIXME : port to cmake
```

Comment:
❓

Reply:
this is rremnant of qt 5 recipe, and these command line arguments need to be ported to cmake definitions, I'll take a look at this later today